src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -99,7 +100,7 @@
 			<NcAppSettingsSection v-if="canLeaveConversation || canDeleteConversation"
 				id="dangerzone"
 				:title="t('spreed', 'Danger zone')">
-				<LockingSettings :token="token" />
+				<LockingSettings v-if="canFullModerate && !isNoteToSelf" :token="token" />
```

Comment:
Pls check that this component doesn't have `canFullModerate` check inside

Reply:
Negative